components/DetailPullup.tsx

```diff
@@ -42,11 +45,21 @@ export function DetailPullUpBody({ item }: { item: Item }) {
         )}
         keyExtractor={(item, index) => index.toString()}
       ></FlatList>
-    </View>
+      <View style={{ height: 130 }}>
```

Comment:
fine for now but two things:
* instead of adding a view for padding you should just add `marginBottom` to the scrollview
* should use styles instead of putting css directly here